### DISCUSSION TITLE
issue while compounding paths

### DISCUSSION THREAD

**Discussion by namrata18s** [simple-icons/simple-icons#10012] [Help with SVGs] [answered]
I am using Inkspace tool to edit svg and while following the steps I am getting stuck in step 2 and 3 -
Merge any overlapping paths.
Compound all paths into one.

whenever I combine paths I am getting a blank canvas. can anyone help me with this? and I even tried to make directions of the two path consistent. after that I got some outline of the shape but if I select other place the outline disappears

**Comment by PeterShaggyNoble** [comment, marked as answer]
This still catches me out every now and again but the first thing to check is that Inkscape isn't changing the `fill` to white (or your canvas' background colour) when combining the paths.

**Reply by adamrusted** [reply]
This, and I also have a few occasions where the _order_ of the paths in my editor messes up how the pathfinder tools work. Some list layers top->bottom as you'd expect to see them, others reverse it - putting the upper-most layer at the bottom of the list!
